tests/testhttp.nim

```diff
@@ -1,76 +1,15 @@
 import unittest, json, strutils
-import httputils, chronicles
+import httputils
 import ../json_rpc/[rpcserver, rpcclient]
 
-const
-  TestsCount = 100
-  BufferSize = 8192
-  BigHeaderSize = 8 * 1024 + 1
-  BigBodySize = 128 * 1024 + 1
-  HeadersMark = @[byte(0x0D), byte(0x0A), byte(0x0D), byte(0x0A)]
+const TestsCount = 100
 
-  Requests = [
```

Comment:
we're not testing HTTP Server + Client anymore, we just assume that's done in `nim-chronos`